command-server/src/main/kotlin/hsenasilva/com/github/sample/cqrs/web/CommandController.kt

```diff
@@ -1,38 +1,50 @@
 package hsenasilva.com.github.sample.cqrs.web
 
-import hsenasilva.com.github.sample.cqrs.core.domain.SampleId
-import hsenasilva.com.github.sample.cqrs.domain.CreateSampleCommand
+import hsenasilva.com.github.sample.cqrs.core.domain.Account
+import hsenasilva.com.github.sample.cqrs.domain.CreditBalanceCommand
+import hsenasilva.com.github.sample.cqrs.domain.DebitBalanceCommand
 import hsenasilva.com.github.sample.cqrs.web.callback.CommandGatewayCallback
-import hsenasilva.com.github.sample.cqrs.web.parameters.CreateSampleParameter
-import hsenasilva.com.github.sample.cqrs.web.parameters.SampleParameter
+import hsenasilva.com.github.sample.cqrs.web.parameters.AmountParameter
 import org.axonframework.commandhandling.gateway.CommandGateway
 import org.springframework.http.HttpStatus
 import org.springframework.web.bind.annotation.*
+import org.springframework.web.server.ResponseStatusException
 import javax.validation.Valid
 import javax.validation.constraints.NotNull
 
 /**
  * @author hsena
  */
 @RestController
-@RequestMapping(value = ["/samples"])
-class CommandController(private val commandGateway: CommandGateway,
-                        private val commandGatewayCallback: CommandGatewayCallback) {
+@RequestMapping(value = ["/commands"])
+class CommandController(
```

Comment:
Could you change the class name `CommandController` and the resource url `/commands` to `CheckingAccountController` and `/checking-accounts` respectively?

Reply:
done